eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -105,6 +105,12 @@ class Peer(val nodeParams: NodeParams, remoteNodeId: PublicKey, wallet: EclairWa
         d.peerConnection forward msg
         stay
 
+      case Event(warning: Warning, _: ConnectedData) =>
+        log.warning("peer sent warning: channelId={} message={}", warning.channelId, warning.toAscii)
```

Comment:
The `channel_id` will be in the MDC:

```suggestion
        log.warning("peer sent warning: message={}", warning.channelId, warning.toAscii)
```

Reply:
Good catch, fixed in https://github.com/ACINQ/eclair/pull/1854/commits/ab67e7ae956ac80267dcc80506559e9a17e48bc5